app/src/features/Batches/BatchList.tsx

```diff
@@ -19,68 +19,75 @@ import BatchCancel from "./BatchCancel";
 import BatchErrors from "./BatchErrors";
 
 const useStyles = makeStyles((theme) => ({
+  accordions: {
+    margin: theme.spacing(0, 5, 8, 5),
+  },
+  accordion: {
+    borderRadius: theme.shape.borderRadius,
+    marginBottom: theme.spacing(0.5),
+    "&.Mui-expanded": {
+      margin: 0,
+      marginBottom: theme.spacing(0.5),
```

Comment:
You could merge those 2 lines into 1 

```suggestion
      margin: theme.spacing(0, 0, 0.5, 0),
```